lib/widgets/hexagon_grid.dart

```diff
@@ -13,14 +13,12 @@ import '../core/player_status.dart';
 class HexagonGrid extends StatefulWidget {
   final BlockGrid grid;
   final PlanetTheme currentTheme;
-  final String type;
```

Comment:
I removed it, but I think that in these cases it is better to use an enum instead string: when you type a string it is possible to do some typing error, enum has a compiling check instead.